hugo/content/8_release-updates/2022.md

```diff
@@ -9,6 +9,14 @@ This section of the document gives you an overview of the new features and funct
 
 # December
 
+## December 28
+
+### Bug Fix
+
+* There was a glitch when a newly invited user signs up using Google Sign-Up. This issue is now fixed.
+* Component executable command gets overridden by cloud params sync logic. This issue is now fixed.
```

Comment:
@pri-lk Can we club the second and third points under a common section for cloud params category?